extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -467,7 +467,10 @@ export class AmpConsent extends AMP.BaseElement {
           return Promise.resolve(consentRequired);
         }
         return this.getConsentRemote_().then(consentResponse => {
-          return !!consentResponse['consentRequired'];
+          return (
+            !!consentResponse['consentRequired'] ||
```

Comment:
we should only use `promptIfUnknown` when `consentRequired = undefined`

Reply:
```js
let consentRequired = consentResponse['consentRequired'];
if (consentRequired === undefined) {
  consentRequired = !!consentResponse['promptIfUnknown'];
}
return consentRequired;
```

Reply:
done